### PR TITLE
Update links to point to TestNet

### PR DESCRIPTION
**Pull request by potatodepaulo**

### CODE REVIEW

**Comment by potatodepaulo** [comment]
Looks like some links are broken... Fixing now.

File: docs/api/rpc-params.md [hunk 1]
```diff
@@ -35,12 +35,12 @@ The following identifiers are used throughout the code samples below.
 
 *At time of writing, `finality` is only available on StageNet.*
 
-To run the examples below, you will likely need a more recent block height or block hash.  You can find that here: https://staging-rpc.nearprotocol.com/status
+To run the examples below, you will likely need a more recent block height or block hash.  You can find that here: https://staging-rpc.testnet.nearprotocol.com/status
```
Comment (frol) [inline docs/api/rpc-params.md:38]: These `staging-rpc.testnet.nearprotocol.com` should be `rpc.testnet.nearprotocol.com`

Comment (vgrichina) [reply docs/api/rpc-params.md:38]: I think it should be `rpc.betanet.nearprotocol.com` 
   
@potatodepaulo  pretty sure `staging-rpc.testnet.nearprotocol.com` doesn't even exist as domain

File: docs/quick-start/create-near-app.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Setup a NEAR Protocol web app template using one command, `npx create-near-app`.
 npx create-near-app my-awesome-project
 cd my-awesome-project
 yarn
```
Comment (vgrichina) [inline docs/quick-start/create-near-app.md:14]: this one is optional now (is done automatically when project is created)

File: docs/api/rpc-params.md (hunk 1 above)
Comment (vgrichina) [reply docs/api/rpc-params.md:38]: `rpc.testnet.nearprotocol.com` does exist though. It depends on context which one we want (testing bleeding edge stuff vs just running apps on testnet).

Comment (potatodepaulo) [reply docs/api/rpc-params.md:38]: yep, got these changes locally already

**Comment by potatodepaulo** [comment]
CI will fail because many of these links are not actually broken. Lots of 429 issues with github links, and anything that shows where a local server is running is read as broken by the link checker. We should make it ignore anything that starts with localhost.

**Comment by amgando** [comment]
force pushed after rebasing on master to remove merge conflicts

File: docs/quick-start/create-near-app.md (hunk 2 above)
Comment (amgando) [reply docs/quick-start/create-near-app.md:14]: ✅

File: docs/api/rpc-params.md (hunk 1 above)
Comment (amgando) [reply docs/api/rpc-params.md]: ✅

**Pull request merged**